src/components/container/LabResultsBloodType/LabResultsBloodType.tsx

```diff
@@ -0,0 +1,149 @@
+import MyDataHelps from "@careevolution/mydatahelps-js";
+import React, { useEffect, useState } from "react";
+import { Action, LoadingIndicator } from "../../presentational";
+import "./LabResultsBloodType.css";
+import "../HealthPreviewSection/HealthPreviewSection.css"
+import language from "../../../helpers/language";
+import { bloodTypeLabs, singleLabs, manyLabs } from "./LabResultsBloodType.previewdata";
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons";
+import iconBloodType from "./icon-bloodtype.svg";
+
+export interface LabResultsBloodTypeProps {
+    previewState?: "BloodTypeLabs" | "SingleLabs" | "ManyLabs" | "NoData"
+    summaryResultsOnly?: true | false
+    maximumResults?: number
+    onClick?: any
+    externalInfoLink?: true | false
+    externalInfoLinkDefault?: string
+    externalInfoLinkSpanish?: string
+    innerRef?: React.Ref<HTMLDivElement>
+}
+
+export default function (props: LabResultsBloodTypeProps) {
+    const [model, setModel] = useState<any>(null);
```

Comment:
I know I used "any" here because I think I was in a rush to get the H&W dashboard out for some use case; but I would take the time to make this be a declared type so intellisense etc. works

Reply:
Fixed